bin/list-packages.rb

```diff
@@ -17,9 +17,7 @@ def list_packages(path,glob,rel)
         debs = [];
         sftp.dir.glob(path,glob) { |f| debs << f }
         
-        p=/^[^_-]+/
-        
-        debs.sort {|x,y| y.name.gsub(p,"")<=>x.name.gsub(p,"")}.each do |f|
+        debs.sort {|x,y| Time.at(y.attributes.mtime)<=>Time.at(x.attributes.mtime)}.each do |f|
```

Comment:
Comparing the versions would be simple using:

```
 Gem::Version.new(a) <=> Gem::Version.new(b
```

which is available in the machine this scripts are executed, so all good.

However, if you ask me, this implies extracting the version number from the `name` using a regex and that may be fragile if the package naming strategy changes for some reason. I still think that ordering by date is the correct, most simple and logic approach.

I will implement the suggested approach anyway so that it can be discussed.